### ISSUE TITLE
Site is down.

### ISSUE THREAD

**Issue by aaronliu0130** [Not-Quite-RARBG/main#50]
### Current Behavior

Visiting <https://nq-rarbg.to/> you get cloudflare's site down message with error 521.

### Expected Behavior

I be able to access site

### Steps To Reproduce

1. go to <https://nq-rarbg.to/>

### Environment

```markdown
- **OS**: Windows 16 (according to [this](https://i.redd.it/1pstidr8d9c91.png))
- **Browser**: Waterfox
```

**Comment by TerrenceTingleberry** [comment]
The reason the site is down is because it's still a work in progress. You're getting the site down error because the site isn't meant to be up yet.

**Comment by aaronliu0130** [comment]
Hmm, thanks! Maybe that should be reflected somewhere, but I just saw a PR on this. On an unrelated note, if The RARBG was forked while this project was still licensed under GPL, why does the README say that's against the license?

**Comment by TerrenceTingleberry** [comment]
> Hmm, thanks! Maybe that should be reflected somewhere, but I just saw a PR on this. On an unrelated note, if The RARBG was forked while this project was still licensed under GPL, why does the README say that's against the license?

We'll update the README in order to reflect this ASAP. Regarding the license disputes with TheRarBg's fork, we've had discussions with their team and are currently working to resolve the dispute.

**Comment by Da-Wit** [comment]
are you guys are still here? it's still down

**Comment by AnonymousWP** [comment]
> are you guys are still here? it's still down

Yeah, we've made lots of progress behind the scenes, [see the commits in the dev branch](https://github.com/Not-Quite-RARBG/main/commits/dev). It should launch this week, but has postponed a few times due to the delay of a few components some devs have been working on.

**Comment by Da-Wit** [comment]
i don't know what's goin on behind but i guess you guys are now losing users to other rarbg alternatives.

**Comment by Da-Wit** [comment]
i meant, A LOT of USERS!

**Comment by aaronliu0130** [comment]
Heh, saying we're losing users is like saying pirating is stealing.

**Comment by AnonymousWP** [comment]
I understand the frustration and confusion, but the reason for that is that some team members have left and that the remaining part is working on private stuff for the time being. We expect to be working on NQ-RARBG next week onwards.

> i don't know what's goin on behind but i guess you guys are now losing users to other rarbg alternatives.

That doesn't matter; what matters is that RARBG is going to be revived, doesn't matter by whom. It's just great to have (legit) choices/alternatives.

**Comment by Disa-Kizonda** [comment]
F

**Comment by zhuyuqian** [comment]
It's almost August

**Comment by zhuyuqian** [comment]
It's almost August，site is still dead...

**Comment by AnonymousWP** [comment]
> It's almost August，site is still dead...

Yes, lot of people left the team due to private reasons. The only developer is currently busy, but the plan is to still launch this project. Later, we're thinking of just keeping this repo to share code on so that everyone can still host a RARBG clone themselves.
